docs/sources/enterprise/usage-insights.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Usage-insights"
+description = "Usage-insights"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/"]
+type = "docs"
+[menu.docs]
+name = "Usage-insights"
+parent = "enterprise"
+weight = 700
++++
+
+# Usage Insights
+
+Usage Insights allows you to have a better understanding on how your Grafana instance is used. The collected data are: dashboard views (aggregated and per user), data source errors, and number of data source queries.
+
+> Only available in Grafana Enterprise v7.0+.
+
+## Presence Indicators
```

Comment:
```suggestion
## Presence indicators
```

Reply:
Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md), headings should be in sentence case. That is why the official name and UI capitalization of "Usage Insights" matters.